assets/section-featured-product.css

```diff
@@ -4,8 +4,9 @@
   padding-bottom: 0;
 }
 
-.featured-product .product__modal-opener {
+.featured-product .product-media-container {
   margin-bottom: var(--media-shadow-vertical-offset);
+  max-width: 100%;
```

Comment:
It's to override this in the main-product styles `.product-media-container { max-width: calc(100% - calc(var(--media-border-width) * 2)) }` https://screenshot.click/13-43-kvtdi-bwbhm.png Funny, I actually did have the thought to comment it, but ultimately assumed it wasn't worth it. Any opinion? It's the kind of thing that if this style was in main product next to other rule it would be plenty clear without the comment, but being in another stylesheet it could be warranted.

Reply:
Ah I see. Since it's a single image. But it's only needed on mobile to overwrite that eh ? But I guess there is no harm in applying it for desktop rather than adding a new media query there. 👍 
I'm cool with it

Reply:
Yeah exactly, I didn't see it as worth adding a mobile only rule or a desktop rule to override it.